.scalafix.conf

```diff
@@ -0,0 +1,5 @@
+ExplicitResultTypes.unsafeShortenNames = true
+
+NoInfer.symbols = [
+  "scala.Predef.any2stringadd"
+]
```

Comment:
Be warned of the caveat in [the docs]—this completely overwrites defaults like `Any` and `Product` that are probably desirable.

[the docs]: https://scalacenter.github.io/scalafix/docs/rules/NoInfer

Reply:
Damn, so I just turned it all off? Oh well, at least it's all wired up now.